### PR TITLE
Add org.sugarlabs.Calculate

### PR DESCRIPTION
**Pull request by SudoSu-bham**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
It is a graphing calculator, which can get the answer to a quick problem, but that is not the limit! we can also explore Algebra, Trigonometry, Boolean and more!

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by bbhtt** [comment]
cc @tchx84

bot, build org.sugarlabs.Calculate

**Comment by tchx84** [comment]
@SudoSu-bham the PR looks OK but, I suspect there's something missing in terms of screen dimension measurements. If I run the application 1280x720 it doesn't fit in the screen and ends up like this. Can you double check this?

Screenshot from 2024-07-30 07-47-35

**Comment by SudoSu-bham** [comment]
@tchx84 Please review the changes.

File: calculate-info.patch [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
++++ b/activity/activity.info
+@@ -1,12 +1,19 @@
+ [Activity]
+ name = Calculate
+-bundle_id = org.laptop.Calculate
++bundle_id = org.sugarlabs.Calculate
+ icon = calculate
+ exec = sugar-activity3 calculate.Calculate -s
+ activity_version = 47
+-license = GPLv2+
```
Comment (tchx84) [inline calculate-info.patch:13]: Keep the license please, see https://github.com/sugarlabs/calculate-activity/blob/master/COPYING

**Review by tchx84 — APPROVED** [review]

**Review by tchx84 — CHANGES_REQUESTED** [review]
See license comment

**Comment by tchx84** [comment]
bot, build org.sugarlabs.Calculate

**Comment by tchx84** [comment]
bot, build org.sugarlabs.Calculate

File: calculate-info.patch [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
++bundle_id = org.sugarlabs.Calculate
+ icon = calculate
+ exec = sugar-activity3 calculate.Calculate -s
+ activity_version = 47
+ license = GPLv2+
+-max_participants = 10
+-summary = This is the place to get the answer to a quick problem, but that is not the limit! You can also explore Algebra, Trigonometry, Boolean and more!
++metadata_license = CC0-1.0
++max_participants = 1
++summary = It is a graphing calculator
```
Comment (tchx84) [inline calculate-info.patch:18]: We need a better summary here, like "An infix-notation graphing calculator" ?

File: calculate-port.patch [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+         self._message_cbs = {}
+diff --git a/toolbars.py b/toolbars.py
+index ca24c2c..9924a00 100644
+--- a/toolbars.py
++++ b/toolbars.py
+@@ -13,6 +13,8 @@ from sugar3.graphics.toolbutton import ToolButton
+ from sugar3.graphics.toggletoolbutton import ToggleToolButton
+ from sugar3.graphics.style import GRID_CELL_SIZE
+ 
++from sugarapp.helpers import PrimaryMonitor
```
Comment (tchx84) [inline calculate-port.patch:55]: Place this in a separate `*-monitors.patch` like we do with all the other apps.

**Review by tchx84 — CHANGES_REQUESTED** [review]

**Comment by tchx84** [comment]
bot, build org.sugarlabs.Calculate

**Review by tchx84 — APPROVED** [review]

**Comment by tchx84** [comment]
@SudoSu-bham looks great now, thanks! CC @bbhtt

**Pull request closed without merging**